po/R-zh_CN.po

```diff
@@ -13,256 +13,256 @@ msgstr ""
 msgid "data contributor"
-msgstr ""
+msgstr "数据提供者"
 
 #: build-home-authors.R:207
 msgid "funder"
-msgstr ""
+msgstr "赞助者"
 
 #: build-home-authors.R:208
 msgid "reviewer"
-msgstr ""
+msgstr "审核者"
 
 #: build-home-authors.R:209
 msgid "thesis advisor"
-msgstr ""
+msgstr "论文导师"
 
 #: build-home-authors.R:210
 msgid "translator"
-msgstr ""
+msgstr "翻译者"
 
 #: build-home-citation.R:39
 msgid "Citing %s"
-msgstr ""
+msgstr "引用 %s"
 
 #: build-home-community.R:18
 msgid "Contributing guide"
-msgstr ""
+msgstr "开发指南"
 
 #: build-home-community.R:22
 msgid "Code of conduct"
-msgstr ""
+msgstr "行为准则"
 
 #: build-home-community.R:29
 msgid "Community"
-msgstr ""
+msgstr "社区"
 
 #: build-home-index.R:85
 msgid "Dev Status"
-msgstr ""
+msgstr "开发状态"
 
 #: build-home-index.R:153
 msgid "View on %s"
-msgstr ""
+msgstr "访问%s"
 
 #: build-home-index.R:154
 msgid "Browse source code"
-msgstr ""
+msgstr "流览源代码"
 
 #: build-home-index.R:155
 msgid "Report a bug"
-msgstr ""
+msgstr "缺陷报告"
 
 #: build-home-index.R:159
 msgid "Links"
-msgstr ""
+msgstr "链接"
 
 #: build-home-index.R:164
 msgid "Table of contents"
-msgstr ""
+msgstr "目录"
 
 #: build-home-license.R:12 build-home-license.R:34
 msgid "License"
-msgstr ""
+msgstr "许可"
 
 #: build-home-license.R:29
 msgid "Full license"
-msgstr ""
+msgstr "许可全部条款"
 
 #: build-news.R:100 build-news.R:233 build-news.R:238
 msgid "Changelog"
-msgstr ""
+msgstr "更改日志"
 
 #: build-news.R:124
 msgid "Version %s"
-msgstr ""
+msgstr "%s 版"
 
 #: build-news.R:136 build-news.R:227
 msgid "News"
-msgstr ""
+msgstr "新闻"
 
 #: build-news.R:229
 msgid "Releases"
-msgstr ""
+msgstr "发布版本"
 
 #: build-news.R:285
 msgid "CRAN release: %s"
-msgstr ""
+msgstr "CRAN发布版本：%s"
 
 #: build-reference-index.R:19
 msgid "Function reference"
-msgstr ""
+msgstr "函数参考"
 
 #: build-tutorials.R:46 navbar.R:100
 msgid "Tutorials"
-msgstr ""
+msgstr "教程"
 
 #: development.R:20
 msgid "Released version"
-msgstr ""
+msgstr "已发布的版本"
 
 #: development.R:21
 msgid "In-development version"
-msgstr ""
+msgstr "开发中的版本"
 
 #: development.R:22
 msgid "Unreleased version"
-msgstr ""
+msgstr "未发布的版本"
 
 #: navbar.R:97
 msgid "Reference"
-msgstr ""
+msgstr "参考"
 
 #: navbar.R:128
 msgid "Get started"
-msgstr ""
+msgstr "入门参考"
 
 #: navbar.R:154
 msgid "More articles..."
-msgstr ""
+msgstr "更多文章…"
 
 #: rd-data.R:33
 msgid "Details"
-msgstr ""
+msgstr "细节"
 
 #: rd-data.R:37
 msgid "Description"
-msgstr ""
+msgstr "概述"
 
 #: rd-data.R:41
 msgid "References"
-msgstr ""
+msgstr "参考文献"
 
 #: rd-data.R:45 render.R:124
 msgid "Source"
-msgstr ""
+msgstr "参考文献出处"
 
 #: rd-data.R:49
 msgid "Format"
-msgstr ""
+msgstr "格式"
 
 #: rd-data.R:53
 msgid "Note"
-msgstr ""
+msgstr "注释"
 
 #: rd-data.R:57
 msgid "Author"
-msgstr ""
+msgstr "作者"
 
 #: rd-data.R:61
 msgid "See also"
-msgstr ""
+msgstr "相关话题"
 
 #: rd-data.R:77
 msgid "Arguments"
-msgstr ""
+msgstr "参数"
 
 #: rd-data.R:85
 msgid "Value"
-msgstr ""
+msgstr "值"
 
 #: rd-html.R:205
 msgid "# %s method for %s"
```

Comment:
This translation could have been easier and less awkward if those 2 were glue strings instead of `%s`s (e.g., `"{method_name} method for {class_name}"`) because in written Chinese it is just way more natural to mention the `{class_name}` first.

The current form is still understandable and within reason though.

Reply:
Oh I should've said that you can change the order — `%2$s` will refer to the second orignal string

Reply:
Alright. This one has been updated to mention the class name first (i.e., the more "natural" order of describing these things in Chinese :slightly_smiling_face: ).

Reply:
Perfect, thanks!